### PR TITLE
[3.8.2] Fix iOS HEADER_SEARCH_PATHS

### PR DESCRIPTION
**Pull request by dmytrozavgorodniy**

### CODE REVIEW

File: scripts/local-ci.sh [hunk 1]
```diff
@@ -97,6 +98,12 @@ rm -rf node_modules && npm install
 # Link project
 react-native unlink $library_name
 react-native link
+
+# Install iOS dependencies
+isMacOS && cd ./ios
+isMacOS && pod install
+isMacOS && cd ../
```
Comment (isnifer) [inline scripts/local-ci.sh]:
```bash
if isMacOS; then
  cd ios
  pod install
  cd ../
fi
```

Comment (dmytrozavgorodniy) [reply scripts/local-ci.sh]: Merci, I will change this snippet

File: .travis/install.sh [hunk 2]
```diff
@@ -4,7 +4,17 @@ library_name=$(node -p "require('./package.json').name")
 
 cd example_tmp
 rm -rf node_modules && npm install
+
 react-native unlink $library_name
 react-native link
+
+case "${TRAVIS_OS_NAME}" in
+  osx)
+    cd ./ios
```
Comment (isnifer) [inline .travis/install.sh]: cd ios

**Comment by brianhiss** [comment]
@dmytrozavgorodniy I've updated my setup below:

```
yarn add https://github.com/tipsi/tipsi-stripe.git#feature/fix-ios-header-search-paths
react-native link tipsi-stripe
```

I've confirmed the changes are in my project and it's compiling, but I'll update back later today if I see any prior issues I was seeing adding and removing `npm` packages.

**Comment by dmytrozavgorodniy** [comment]
We already checked this fix in our app and it work for us. Waiting for your feedback.

**Comment by brianhiss** [comment]
@dmytrozavgorodniy So far so good. Been running a day adding and removing packages, cleaning the build and compiling without conflicts.

Looks like you might be good to go. I'll let you know if I come across anything.

Thank you for fixing that.

**Comment by cybergrind** [comment]
Thank you all!

**Pull request merged**